source/_integrations/tesla_wall_connector.markdown

```diff
@@ -21,3 +21,37 @@ ha_integration_type: integration
 The Tesla Wall Connector integration allows you to integrate your Gen 3 [Tesla Wall Connector](https://www.tesla.com/support/home-charging-installation/wall-connector) with Wi-Fi into Home Assistant.
 
 {% include integrations/config_flow.md %}
+
+## Entities
```

Comment:
We have a standardized way of documenting this, see: <https://developers.home-assistant.io/docs/core/integration-quality-scale/rules/docs-supported-functions>

Could you please adjust the PR to match this?